k8s/airqo-deployments.yaml

```diff
@@ -0,0 +1,160 @@
+###########################################################
+# Start deployment of authentication service microservice
+###########################################################
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: airqo-auth-service
+  namespace: staging
+spec:
+  selector:
+    matchLabels:
+      app: auth-service
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: auth-service
+    spec:
+      containers:
+        - name: airqo-auth-service
+          image: us.gcr.io/airqo-250220/airqo-auth-service:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 3000
+              name: airqo-auth-service
+
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: airqo-auth-service
+  namespace: staging
+spec:
+  selector:
+    app: auth-service
+  ports:
+    - protocol: TCP
+      port: 3000
+      targetPort: 3000
+      nodePort: 30001
+  type: NodePort
+
+##################################################################
+# Start data management microserive deployment and provisioning
+##################################################################
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: airqo-datamgt-service
+  namespace: staging
+  labels:
+    app: airqo-datamgt-service
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: airqo-datamgt-service
+    spec:
+      containers:
+        - name: airqo-datamgt-service
+          image: us.gcr.io/airqo-250220/airqo-datamgt-service:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 3000
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: airqo-datamgt-service-svc
```

Comment:
A renaming suggestion: **airqo-data-mgt-service**. 

How does this sound?

Reply:
Sounds good. Comment implemented